Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM python:3.9
 
 WORKDIR /app
 
-COPY requirements /app/requirements
+COPY requirements.txt /app/requirements.txt
 
 RUN pip install -r requirements.txt
 COPY . /app
 
-RUN start.sh
\ No newline at end of file
+CMD bahs start_server.sh
```

Comment:
Spelling error for the bash. 

```
RUN bash start_server.sh
```
```
CMD bahs start_server.sh
```

I am not sure about cmd, cmd is for windows only commands right? will it work on Docker?

Reply:
Done